command/file.go

```diff
@@ -33,24 +33,28 @@ func CopyDir(src, dst string, isOnlyContent bool) error {
 }
 
 // RemoveDir ...
-func RemoveDir(dirPth string) error {
-	if exist, err := pathutil.IsPathExists(dirPth); err != nil {
-		return err
-	} else if exist {
-		if err := os.RemoveAll(dirPth); err != nil {
+func RemoveDir(dirPths ...string) error {
+	for _, dirPth := range dirPths {
+		if exist, err := pathutil.IsPathExists(dirPth); err != nil {
 			return err
+		} else if exist {
+			if err := os.RemoveAll(dirPth); err != nil {
+				return err
+			}
 		}
 	}
 	return nil
```

Comment:
except the 'for' it is not necessary to have a single os.RemoveAll command call as it is not an optimization